test/metrics/basic_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'browser_helper'
+
+describe :metrics, :js do
+  describe :basic do
+    after do
+      puts "Cleaning up test series and episodes" if debug?
+      delete_test_series_and_episodes!
+    end
+
+    it 'registers when podcast is played' do
+      random_str = SecureRandom.hex(10)
+      mp3 = setup_fixtures(random_str)
```

Comment:
I'm getting an error on this line locally-- doesn't seem to be waiting long enough (or possible I have something configured poorly locally?). Anyone else having this? 
```ruby
Capybara::ElementNotFound:         Capybara::ElementNotFound: Unable to find visible css "div.success"
            /Users/gigiblum-campo/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/capybara-2.15.4/lib/capybara/node/finders.rb:314:in `block in synced_resolve'
            /Users/gigiblum-campo/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/capybara-2.15.4/lib/capybara/node/base.rb:85:in `synchronize'
            /Users/gigiblum-campo/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/capybara-2.15.4/lib/capybara/node/finders.rb:302:in `synced_resolve'
            /Users/gigiblum-campo/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/capybara-2.15.4/lib/capybara/node/finders.rb:37:in `find'
            /Users/gigiblum-campo/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/capybara-2.15.4/lib/capybara/session.rb:792:in `block (2 levels) in <class:Session>'
            /Users/gigiblum-campo/.rbenv/versions/2.3.1/lib/ruby/gems/2.3.0/gems/capybara-2.15.4/lib/capybara/dsl.rb:50:in `block (2 levels) in <module:DSL>'
            /Users/gigiblum-campo/Desktop/PRX_repos/more repos/meta.prx.org/test/support/publish_dsl.rb:44:in `publish_wait_for_success'
            /Users/gigiblum-campo/Desktop/PRX_repos/more repos/meta.prx.org/test/support/publish_dsl.rb:164:in `create_series!'
            /Users/gigiblum-campo/Desktop/PRX_repos/more repos/meta.prx.org/test/metrics/basic_test.rb:34:in `setup_fixtures'
            /Users/gigiblum-campo/Desktop/PRX_repos/more repos/meta.prx.org/test/metrics/basic_test.rb:12:in `block (3 levels) in <top (required)>'

```

Reply:
So one thought i had on your `publish_fetch_rss_media_url` (which waits for feeder to upload the RSS to S3)...

It's a bit less integration-y, but you could also check some less-cache-y feeder endpoints for the episode:

1. https://f-staging.prxu.org/213/feed-rss.xml (slow, but it's what Publish tells you about directly)
2. https://feeder.staging.prx.tech/podcasts/213 (exactly the same as above, but updates immediately)
3. https://feeder.staging.prx.tech/api/v1/podcasts/213/episodes (if you don't like xml)

I almost think (2) is the most viable, since it's literally the same as what Feeder copies over to S3.

Reply:
ah cool. I had been using (1) in the tests, but will switch to (2).

Reply:
So in my experiments, (2) isn't really much faster than (1) because the `item` doesn't appear until some background task finishes. Still, going to use (2) to avoid the S3 step.